Firestore/core/src/core/query.h

```diff
@@ -244,6 +244,12 @@ class Query {
    */
   const Target& ToTarget() const&;
 
+  /**
+   * Returns a `Target` instance this query will be mapped to in backend
+   * and local store, for use within an aggregate query.
```

Comment:
It's probably worth adding a comment about how this is different from ToTarget(). e.g.

```
  /**
   * Returns a `Target` instance this query will be mapped to in backend
   * and local store, for use within an aggregate query. Aggregate query targets
   * do not contain normalized order-bys and only contain explicit order-bys.
   * /
```

Reply:
Updated in both locations. Thanks for pointing this out.